apps/pharos_web/mix.exs

```diff
@@ -32,14 +32,16 @@ defmodule PharosWeb.Mixfile do
   #
   # Type `mix help deps` for examples and options.
   defp deps do
-    [{:phoenix, "~> 1.2.1"},
-     {:phoenix_pubsub, "~> 1.0"},
-     {:phoenix_html, "~> 2.6"},
-     {:phoenix_live_reload, "~> 1.0", only: :dev},
-     {:gettext, "~> 0.11"},
-     {:cowboy, "~> 1.0"},
-     {:post_service, in_umbrella: true},
-     {:memory_db, in_umbrella: true}
-     ]
+    [
+      {:phoenix, "~> 1.2.1"},
+      {:phoenix_pubsub, "~> 1.0"},
+      {:phoenix_html, "~> 2.6"},
+      {:phoenix_live_reload, "~> 1.0", only: :dev},
+      {:gettext, "~> 0.11"},
+      {:cowboy, "~> 1.0"},
+      {:post_service, in_umbrella: true},
+      {:memory_db, in_umbrella: true},
+      {:twitter, in_umbrella: true}
```

Comment:
We have the memory_db and the post_service so I think we can leave that out for them to put in 👍